src/main.c

```diff
@@ -60,19 +57,9 @@ static void main_log (hashcat_ctx_t *hashcat_ctx, FILE *fp, const int loglevel)
 
   const size_t prev_len = event_ctx->prev_len;
 
-  if (prev_len)
-  {
-    main_log_clear_line (prev_len, fp);
-  }
+  if (prev_len) main_log_clear_line (prev_len, fp);
 
-  if (msg_newline == true)
-  {
-    event_ctx->prev_len = 0;
-  }
-  else
-  {
-    event_ctx->prev_len = msg_len;
-  }
+  event_ctx->prev_len = (msg_newline == true) ? 0 : msg_len;
```

Comment:
Please go back to old format, it's easier to read in block.

Reply:
I love ternary operator :)
it's used also in other sections but if you prefer I switch back to old format